tasks/transform_conformance/tests/babel-plugin-transform-react-jsx/test/fixtures/misc/arbitrary_length_member_expr/options.json

```diff
@@ -0,0 +1,11 @@
+{
+  "plugins": [
+    [
+      "transform-react-jsx",
+      {
+        "runtime": "classic",
+        "pragma": "a.b.c"
```

Comment:
Before an error will be raised, since before we only support member expr length of two, https://github.com/oxc-project/oxc/blob/6ef9cb06a3b56ee35bdffd1d9487ddb6c8746c73/crates/oxc_transformer/src/jsx/jsx_impl.rs?plain=1#L340-L345